packages/vc-verification/src/Ipfs-credential-resolver.ts

```diff
@@ -31,11 +31,30 @@ export class IpfsCredentialResolver implements CredentialResolver {
    * @returns
    */
   async getCredential(did: string, role: string) {
+    const credentials = await this.credentialsOf(did);
+    return credentials.find(
+      (claim) =>
+        claim.credentialSubject.role.namespace === role ||
+        utils.namehash(claim.credentialSubject.role.namespace) === role
+    );
+  }
+
+  /**
+   * Fethces credential for the given did and role
```

Comment:
The documentation for `getClaimIssuedToken` and `getCredential` are the same which seems a bit confusing as they are different methods. Could be a bit more specific in the documentation please?